iroh-net-report/src/lib.rs

```diff
@@ -232,18 +238,22 @@ pub struct Options {
     /// other packets from in the magicsocket (`MagicSock`).
     ///
     /// If not provided, STUN probes will not be sent over IPv6.
+    #[cfg(not(wasm_browser))]
     stun_sock_v6: Option<Arc<UdpSocket>>,
```

Comment:
maybe create `StunConfig` and `IcmpConfig`, like there is `QuicConfig` to group things a little nicer

Reply:
I've simplified lots of these cfgs by plucking out a wasm-only version of `Options`, instead of doing a bunch of cfgs on the fields & methods.